node/src/lib.rs

```diff
@@ -15,7 +27,156 @@ pub mod peer_manager;
 pub mod proxy_capnp;
 pub mod rpc;
 pub mod rpc_server;
+pub mod stratum;
 pub mod template_creator;
 pub mod uncommitted_metadata;
 pub mod utils;
 pub mod zmq;
+/// **Length of the extranonce prefix (in bytes).**
+///
+/// In Stratum mining, the extranonce is split into two parts:
+/// `EXTRANONCE1` (prefix) and `EXTRANONCE2` (suffix).
```

Comment:
This isn't correct, the assembled coinbase should be (schematically):

coinbase = prefix + extranonce1 + separator + extranonce2 + suffix